docs/source/architecture/events_and_transactions_receipts.rst

```diff
@@ -0,0 +1,341 @@
+*******************************
+Events and Transaction Receipts
+*******************************
+Hyperledger Sawtooth supports the creation and broadcasting of Events.
```

Comment:
"creation and broadcast" - the tenses should probably match

Reply:
Technically, this is the noun form, not (verb) tense, but I second this change. - signed, the Pedant Writer